pyblish_bumpybox/plugins/validate_scene_version.py

```diff
@@ -6,8 +8,11 @@ class ValidateSceneVersion(pyblish.api.ContextPlugin):
 
     order = pyblish.api.ValidatorOrder
     label = "Scene Version"
+    optional = True
 
     def process(self, context):
 
-        msg = "Could not find a version number in the scene name."
-        assert context.has_data("version"), msg
+        name, ext = os.path.splitext(context.data("currentFile"))
+        msg = "Could not find a version number in the scene name. Please add "
+        msg += "v[number] to the scene name, for example: \"{0}_v001{1}\"."
+        assert "version" in context.data, msg.format(name, ext)
```

Comment:
Having the msg.format(name, ext) in the assert statement seems unnecessary here, given it doesn't form part of the assertion.